src/main/java/homework_nr_3/circle.java

```diff
@@ -0,0 +1,15 @@
+package homework_nr_3;
+
+public class circle {
+    public static void main(String[] args){
+        float radius;
+        radius = calculateArea(23);
+        System.out.println(radius);
+
+    }
+
+    public static float calculateArea(float x){
```

Comment:
remove static, we didn't learn it yet. You have to create an object from the circle class